Updating TC_CADMIN_1_9 test module due to issues noticed during step 4

j-ororke opened the pull request:
- Replacing wait with using 3 and 50 as error codes possible when attempting to commission on the 21st attempt due to expecting timeout error but possibly receiving incorrect state as mentioned in issue project-chip/matter-test-scripts#459

j-ororke committed:
Updating TC_CADMIN_1_9 test module:
- Replacing wait with using 3 and 50 as error codes possible when attempting to commission on the 21st attempt!

restyled-commits committed:
Restyled by autopep8

j-ororke committed:
Updating TC_CADMIN_1_9 test module:
- Attempting to resolve linting error for bare except

j-ororke committed:
Updating TC_CADMIN_1_9 test module:
- Attempting to resolve linting errors

cecille commented on src/python_testing/TC_CADMIN_1_9.py:
```diff
@@ -105,7 +105,13 @@ async def CommissionAttempt(
+            try:
+                asserts.assert_true(errcode.sdk_code == expectedErrCode,
+                                    'Unexpected error code returned from CommissioningComplete')
+            except Exception:
```
except Exception is quite broad. Can you change this to just
asserts.assert_in(errcode.sdk_code, [expectedErrCode, 3], 'Unexpected error code returned from CommissioningComplete')

j-ororke committed:
Updating CADMIN_1_9 test module:
- Replaced exception to using asserts.assert_in() function instead, much cleaner!
